localization/english/wc_regions_l_english.yml

```diff
@@ -3,7 +3,7 @@
  world_innovation_camel:0 "Tanaris or Uldum"
  custom_sea_locked:0 "Sea Locked counties"
  custom_middle_kalimdor:0 "Central and Southern Kalimdor"
- custom_desert_regions:0 "Desert Territory"
+ custom_arid_regions:0 "Desert or Drylands Territory"
```

Comment:
Why not simply name it "Drylands" Territory? Deserts are just an extreme of that, aren't they?

Reply:
I think arid and frigid sound better and make more sense since they are more general

Reply:
Hm? No, i meant why call it "Desert or Drylands" instead of just drylands.